### DISCUSSION TITLE
Limit Fields Searched in Grid View Search

### DISCUSSION THREAD

**Discussion by cancan101** [pimcore/pimcore#16927] [Q&A / Support] [answered]
Is there a way to limit / control which fields are searched when using the search on the top of the grid view?
![image](https://github.com/pimcore/pimcore/assets/51059/7e03fe8b-cee6-4916-ad7e-51c4190e9764)

I have some pretty long text fields that I would like to _avoid_ searching as they generate spurious results. I am open to an opt-in or an opt-out solution.

**Comment by markus-moser** [comment, marked as answer]
For some datatypes like textarea or WYSIWYG there is a checkbox "Exclude from Backend Full-Text Search" in the class editor. With this it is possible to exclude the attribute from the search. But after checking these checkboxes it is needed to re-index the backend search as otherwise those fields are still stored in the backend search table. Another benefit of this approach is that the backend search table becomes smaller and therefore the search will become faster. There is a console command available which handles the re-indexing.
